boards/sodaq-explorer/include/board.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-explorer
+ * @brief       Board specific definitions for the SODAQ ExpLoRer board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the SODAQ ExpLoRer board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
with #6702 merge you need to add 

```
/**
 * @name    xtimer configuration
 * @{
 */
#define XTIMER_WIDTH        (16)
/** @} */
```

or use TIMER1 for xtimer with default with of 32 bits

Reply:
I kept the 16 bit version

Reply:
did you run any of the `tests/xtimer_*` to verify, because this must be tested at runtime, otherwise Murdock would've complained already.

Reply:
I tested with `xtimer_msg`, works well.

Reply:
thx for reporting, ACK then

Reply:
xtimer_drift is good test to run for an overall sanity check of the hardware timer. It will show you if the timer is misconfigured, for example running at slower or faster than expected. It should print one message per second if everything is working properly.

Reply:
I ran this one as well and had the expected behaviour ;)